CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing Docs
+
+## Local Dev
```

Comment:
maybe it's just me but I always forget how to do this so I added it to our docs. Idk if this is really best place for it?

Reply:
it's helpful so my +1